.github/workflows/test.yaml

```diff
@@ -35,172 +43,132 @@ jobs:
         with:
           version: v3.10.0
       
+      - name: Copy cloud creds to file
+        run: 'echo "$CLOUD" > clouds.yml'
+        shell: bash
+        env:
+          CLOUD: ${{ secrets.CLOUD }}
+      
       - name: Create k8s Kind Cluster
         uses: helm/kind-action@v1.5.0
       
       - name: Install cert-manager
-        run: |-
-          helm upgrade cert-manager cert-manager \
-            --repo https://charts.jetstack.io \
-            --version v1.10.1 \
-            --namespace cert-manager \
-            --create-namespace  \
-            --install \
-            --set installCRDs=true \
-            --wait \
-            --timeout 5m
-      
-      - name: Ensure Cluster API kustomization directory exists
-        run: mkdir -p clusterapi
-      
-      - name: Write Cluster API kustomization file
-        uses: DamianReeves/write-file-action@master
+        run: >
+          helm repo add jetstack https://charts.jetstack.io
+          && helm repo update 
+          && helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.10.1 --set installCRDs=true --wait
+      
+      - name: Make kustomization directory 
+        run: mkdir clusterapi
+      
+      # From here: https://github.com/stackhpc/ansible-collection-azimuth-ops/blob/main/roles/clusterapi/defaults/main.yml
+      - name: Overwrite file
+        uses: "DamianReeves/write-file-action@master"
         with:
           path: clusterapi/kustomization.yaml
           write-mode: overwrite
           contents: |
```

Comment:
nit: I think all of the formatting changes here actually make this less readable and don't actually change anything